Dockerfile-buildkit

```diff
@@ -0,0 +1,44 @@
+FROM golang:1.16-alpine AS package
```

Comment:
We need to embed the rack build binary into the buildkit image. The other way around doesn't work.